watch css task doesn't work

### Expected Behaviour

CSS watch is possible

### Actual Behaviour

CSS watch fails

### Reproduce Scenario (including but not limited to)

#### Steps to Reproduce

1. Install the FE Build package
2. Add the watch CSS task
3. Run the task

#### Sample Code that illustrates the problem

package.json

```
"watch:css": "nc-fe-build --task=styles --watch"
```

#### Logs taken while reproducing problem

```bash
> nc-fe-build --task=styles --watch

[node_modules/@netcentric/fe-build/tasks/run.js] ☞  looking for other configurations if it finds its runs 
[node_modules/@netcentric/fe-build/tasks/run.js] ⚠   Found local configurations - Running it instead
[node_modules/@netcentric/fe-build/utils/extendConfig.js] ☞  Extending Fe build config for .
[node_modules/@netcentric/fe-build/utils/taskVerification.js] ☞  Tasks started with mode - production
[node_modules/@netcentric/fe-build/tasks/styles.js] ☞  Watcher Sass / autoprefixer running...
[node_modules/@netcentric/fe-build/tasks/styles.js] ✖  Something is missing did you run `ncps fe-install`?Cannot find module 'gaze'
Require stack:
- /Users/ricardtorres/Downloads/build-test/node_modules/@netcentric/fe-build/tasks/styles.js
- /Users/ricardtorres/Downloads/build-test/node_modules/@netcentric/fe-build/utils/taskVerification.js
- /Users/ricardtorres/Downloads/build-test/node_modules/@netcentric/fe-build/tasks/run.js
- /Users/ricardtorres/Downloads/build-test/node_modules/@netcentric/fe-build/cli/index.js
```

--- a/docs/configuration.md
+++ b/docs/configuration.md
@@ -7,24 +7,26 @@ It is used for all files located in the same directory as the `.febuild` file an
 You can add multiple `.febuild` whenever you need to run a separate build with other options.
 
 These are the configurations that can be extended:
--  general
--  output
--  optimization
--  plugins
--  eslint
--  babel
--  sass
--  clientlibs
--  stylelint
--  resolve
--  postcss
--  templates
+
+- general
+- output
+- optimization
+- plugins
+- eslint
+- babel
+- sass
+- clientlibs
+- stylelint
+- resolve
+- postcss
+- templates
 
 ## Overriding a Default Configuration
 
 To override a configuration, add a new entry in the object exported in your `.febuild` with the name of the configuration you want to override, which value is an object whith entries that matches the existing options you want to override.
 
 E.g., to override the default Babel configuration with new `exclude` paths and plugins, you can do the following in your `.febuild` file:
+
 ```javascript
 module.exports = {
     babel: {
@@ -53,13 +55,14 @@ module.exports = (defaultConfig) => ({
 ```
 
 ## Configurations
+
 ### General
 
-This configuration part is used for basic project setup. You will find an explanation of each key as a comment next to it.  
+This configuration part is used for basic project setup. You will find an explanation of each key as a comment next to it.
 Defaults:
 
 ```javascript
-{
+module.exports = {
   general: {
       // Your project name with which ClientLibs category are prefixed
       projectKey: "myproj",
@@ -86,9 +89,12 @@ Defaults:
 ```
 
 Example of overriding the `defaultTasks` to exclude the 'clientlibs' task:
+
 ```javascript
-general: {
-    defaultTasks: ['styles', 'webpack']
+module.exports = {
+  general: {
+      defaultTasks: ['styles', 'webpack']
+  }
 }
 ```
 
@@ -116,7 +122,7 @@ For more information about the configuration options check [babel-loader](https:
 
 #### @babel/preset-env
 
-By default `@babel/preset-env` will use [`browserslist` config sources](https://github.com/browserslist/browserslist) _unless_ either the `targets` or `ignoreBrowserslistConfig` options are set.  
+By default `@babel/preset-env` will use [`browserslist` config sources](https://github.com/browserslist/browserslist) _unless_ either the `targets` or `ignoreBrowserslistConfig` options are set.
 
 > When no browser targets are specified, Babel will assume the oldest browsers possible, which will increase the output code size.
 
@@ -127,8 +133,8 @@ We recommend using a [.browserslistrc](https://github.com/browserslist/browsersl
 This option configures how Babel [handles the polyfills](https://babeljs.io/docs/en/babel-preset-env#usebuiltins), by adding the specific imports only when the polyfill is used in each file.
 
 What is the advantages over "entry"?
-+ It allows proper tree-shaking the polyfills
-+ Reduce the size of the JavaScript file entrypoint
+- It allows proper tree-shaking the polyfills
+- Reduce the size of the JavaScript file entrypoint
 
 #### Core JS 3
 
@@ -143,10 +149,12 @@ Stylelint is a CSS linter which can also lint Scss files.
 Default configuration:
 
 ```javascript
-{
-  stylelint: { 
-    // Stops the build if a linter error is found
-    failOnError: true 
+module.exports = {
+  {
+    stylelint: {
+      // Stops the build if a linter error is found
+      failOnError: true
+    }
   }
 }
 ```
@@ -154,25 +162,27 @@ Default configuration:
 Please note that you need a [Stylelint configuration object]((https://stylelint.io/user-guide/configure/)) to parse CSS-like languages like Scss. We recommend extending a shared configuration like [@netcentric/stylelint-config](https://github.com/Netcentric/stylelint-config).
 
 You can add your own linter rules in the [Stylelint configuration object](https://stylelint.io/user-guide/configure/#rules).
+
 ### ESlint
 
-ESLint statically analyzes your code to quickly find problems. 
+ESLint statically analyzes your code to quickly find problems.
 
 For more information about the configuration options check [eslint-loader](https://github.com/webpack-contrib/eslint-loader).
 
-
 ```javascript
-{
-  eslint: {
-    enforce: 'pre',
-    test: /\.js$/,
-    exclude: /node_modules/,
-    use: {
-      loader: 'eslint-loader',
-      options: {
-        cache: true,
-        failOnError: true,
-        fix: true
+module.exports = {
+  {
+    eslint: {
+      enforce: 'pre',
+      test: /\.js$/,
+      exclude: /node_modules/,
+      use: {
+        loader: 'eslint-loader',
+        options: {
+          cache: true,
+          failOnError: true,
+          fix: true
+        }
       }
     }
   }
@@ -196,16 +206,17 @@ For more information about the configuration options check [eslint-loader](https
 
 ### PostCSS
 
-[PostCSS](https://postcss.org/) is used to transform the CSS code generated after the Sass compilation.  
+[PostCSS](https://postcss.org/) is used to transform the CSS code generated after the Sass compilation.
 
 Default configuration:
+
 ```javascript
 {
   postcss: {
     // Default plugins
     plugins: ['autoprefixer'],
     // Stops the build if an error is found
-    failOnError: true 
+    failOnError: true
   }
 }
 ```
@@ -223,6 +234,7 @@ postcss: {
 This configuration part refers to [Webpack plugins](https://webpack.js.org/plugins/define-plugin/).
 
 Default configuration:
+
 ```javascript
 {
   plugins: [
@@ -279,8 +291,8 @@ Default configuration:
 If a module is imported into more than 2 files, it's extracted to a common file.
 There are 2 main common files:
 
-+ `treeshaking.bundle.js`: common code that reside in the project.
-+ `vendors.bundle.js`: common code that is outside of the scope your project.
+- `treeshaking.bundle.js`: common code that reside in the project.
+- `vendors.bundle.js`: common code that is outside of the scope your project.
 
 **vendors.bundle.js**
 
@@ -288,8 +300,8 @@ This is intended to extract reused third-party scripts that are imported in two 
 Also it is good to separate those third-party from the regular tree shaking since this vendors sometimes might be best suited as an external option.
 
 Advantages of having a separated vendor:
-+ Clear view of the impact of third-party on your code base.
-+ You can identify possible additions to externals (removing it completely from your code).
+- Clear view of the impact of third-party on your code base.
+- You can identify possible additions to externals (removing it completely from your code).
 
 **treeshaking.bundle.js**
 

--- a/docs/migration.md
+++ b/docs/migration.md
@@ -5,11 +5,13 @@ This release contains breaking changes. We know these can be disruptive, but the
 [Stylelint v14](https://stylelint.io/migration-guide/to-14/) does not longer includes the syntaxes that parse CSS-like languages like Scss. You will need to install and configure these syntaxes in your project. We recommend extending a shared configuration like [@netcentric/stylelint-config](https://github.com/Netcentric/stylelint-config) that includes the appropriate syntax to lint Scss.
 
 First, install the shared configuration as a dependency:
+
 ```bash
 npm install --save-dev @netcentric/stylelint-config
 ```
 
 Then, update your [Stylelint configuration object](https://stylelint.io/user-guide/configure/) to use it:
+
 ```json
 {
   "extends": "@netcentric/stylelint-config",

--- a/docs/quick_start.md
+++ b/docs/quick_start.md
@@ -15,6 +15,7 @@ e.g. In your project:
 Note that on the first execution of the `npm run build` task, probably no files will be processed, because no file will match with default settings.
 
 To start the build and change the default settings, add a `.febuild` file one level up from where your `projectSrcDir` directory is with the following content:
+
 ```javascript
 module.exports = {}
 ```
@@ -24,9 +25,11 @@ You can check the default settings for each specific section in the [configurati
 ## Custom Source Path
 
 In order to start processing the files in your project, two updates are needed:
-  1. Add the `source` suffix to all the files that needs to be processed. This suffix value is defined in `general.sourceKey`.  
+
+  1. Add the `source` suffix to all the files that needs to be processed. This suffix value is defined in `general.sourceKey`.
      e.g.: `file.scss` --> `file.source.scss`
   2. In the `.febuild` file, change the source directory `projectSrcDir` configuration to the path to where your source code is.
+
      ```javascript
      module.exports = {
        general: {
@@ -34,9 +37,11 @@ In order to start processing the files in your project, two updates are needed:
        }
      }
      ```
+
      if `sourcePath` is not provided, the path where the `.febuild` file is will be used instead. For this example it will be enough.
 
 After running again the `npm run build` task, this will be the output the destination folder:
+
 ```
 -- package.json
 -- .febuild

--- a/docs/tasks.md
+++ b/docs/tasks.md
@@ -1,14 +1,17 @@
 # Available NPM Tasks
+
 The following npm tasks are available by default after installing fe-build, but you need to add them manually to your package.json file.
 
 ## Compile JavaScript/ECMAScript
 
 Add `nc-fe-build --task=webpack` task in package.json scripts
+
 ```json
   "scripts": {
     "build:js": "nc-fe-build --task=webpack"
   },
 ```
+
 To execute it, open the terminal and run `npm run build:js` from the same folder where the file package.json is.
 
 ## Compile Sass

--- a/package.json
+++ b/package.json
@@ -44,13 +44,13 @@
     "eslint-loader": "4.0.2",
     "eslint-plugin-import": "2.26.0",
     "fast-glob": "3.2.2",
+    "gaze": "^1.1.3",
     "postcss": "8.4.6",
     "sass": "1.44.0",
     "stylelint": "14.6.1",
     "webpack": "4.46.0"
   },
   "devDependencies": {
-    "gaze": "^1.1.3",
     "webpack-bundle-analyzer": "^3.7.0"
   },
   "browserslist": [

